woocommerce/woocommerce
# package maintenance for stylelint vulnerability

### Changes proposed in this Pull Request:

This PR updates the package version on `stylelint` and `stylelint-config-wordpress`. The current version of `stylelint` will no longer install due to a [vulnerability in its dependencies](https://snyk.io/test/npm/stylelint/12.0.1). 

It includes package-lock maintenance which addresses #30011.

Closes #30011 .

### How to test the changes in this Pull Request:

1. Run `npm i` in this branch
2. Verify stylelint will install
```
$ npm list stylelint
woocommerce@5.6.0 /Users/ronrennick/Sites/solaris/woocommerce
└── stylelint@13.13.1 
```

### Changelog entry

N/A
